src/components/AddCommentForm.js

```diff
@@ -0,0 +1,37 @@
+import { useState, useContext } from "react";
+import { UserContext } from "../contexts/UserContext";
+import { addCommentToArticleByID } from "../utils/api";
+
+const AddCommentForm = ({ article_id, setVisibleComments }) => {
+  const [commentText, setCommentText] = useState("");
+  const { signedInUser } = useContext(UserContext);
+
+  console.log(commentText);
+
+  const handleInput = (event) => {
+    setCommentText(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (signedInUser === "") return;
+    addCommentToArticleByID(article_id, signedInUser, commentText).then(
+      (addedComment) => {
+        setVisibleComments((currVisibleComments) => {

```

Comment:
From a UX point - could we reset the form once its successfully posted so my previous text isnt still there?

Reply:
Current the text is still there

Reply:
clears textarea on submit in latest commit